### PR TITLE
f-header@9.15.0 - pie updates

### PR DESCRIPTION
**Pull request by FayeCarter**
### Added
- `@justeattakeaway/pie-icons-vue` icons.
- `Help` icon.
- `Chevron` icon to country selector.

### Changed
- Hamburger and `Gift` icon colours in mobile view.
- Padding and margin changes to accommodate new icons.

## Full width header

|Before | After|
| ---- | --- |
| image| image |
| image | image |
| image |image |

## Mobile header
|Before | After|
| ---- | --- |
| image | image|
| image |  image |
| image  |  image |

## Account popover
|Before | After|
| ---- | --- |
| image| image | 

## Mobile header - Hamburger open
|Before | After|
| ---- | --- |
| image | image |
| image | image |

## Mobile header - Country selector open
|Before | After|
| ---- | --- |
| image | image |
| image | image|

### CODE REVIEW

File: packages/components/organisms/f-header/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+v9.15.0
+------------------------------
+*May 9, 2022*
+
+### Added
+- `@justeattakeaway/pie-icons-vue` icons.
+
+### Changed
+- Hamburger and give icon colours.
+- Small css changes.
```
Comment (ashleynolan) [inline packages/components/organisms/f-header/CHANGELOG.md]: Was the CSS change for the padding on the popover? Would be good to put a bit of detail as to what the change was.

File: packages/components/organisms/f-header/package.json [hunk 2]
```diff
@@ -62,6 +62,7 @@
     "@justeat/f-button": "3.3.0",
     "@justeat/f-popover": "2.3.0",
     "@justeat/f-vue-icons": "3.4.0",
+    "@justeattakeaway/pie-icons-vue": "3.9.0",
```
Comment (ashleynolan) [inline packages/components/organisms/f-header/package.json]: Guessing this is just a placeholder, as the version will be v1.0.0 (can use a beta version to test this atm)

File: packages/components/organisms/f-header/src/components/CountrySelector.vue [hunk 3]
```diff
@@ -27,6 +27,9 @@
                 :class="$style['c-countrySelector-text']">
                 {{ copy.selectYourCountryText }}
             </span>
+            <chevron-right-icon
+                v-if="isBelowMid"
+                :class="$style['c-countrySelector-back-arrow']" />
```
Comment (ashleynolan) [inline packages/components/organisms/f-header/src/components/CountrySelector.vue]:
```suggestion
                :class="$style['c-countrySelector-backIcon']" />
```

File: packages/components/organisms/f-header/src/components/CountrySelector.vue [hunk 4]
```diff
@@ -157,4 +162,9 @@ export default {
         float: left;
     }
 }
+
+.c-countrySelector-back-arrow {
```
Comment (ashleynolan) [inline packages/components/organisms/f-header/src/components/CountrySelector.vue]:
```suggestion
.c-countrySelector-backIcon {
```

File: packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue [hunk 5]
```diff
@@ -157,14 +158,13 @@ $countrySelector-text-bg-hover : $color-container-subtle;
...
-        margin-left: spacing(b);
+        margin: spacing(b) spacing() 0;
+
         @include media('>mid') {
             display: none;
         }
 
-        svg.c-countrySelector-goBackIcon {
-            transform: rotate(180deg);
+        .c-countrySelector-goBackIcon {
```
Comment (ashleynolan) [inline packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue]: What's the difference between this class and the other `-backArrow` class in the other countrySelector component?

Just wondering as they have v similar names atm and makes it a little confusing.

File: packages/components/organisms/f-header/src/components/Navigation.vue [hunk 6]
```diff
@@ -62,7 +62,7 @@
                 $style['c-nav-featureLink--hideAboveMid']
             ]">
             <gift-icon
-                :class="{ [$style['c-nav-icon--alt']]: isAltColour }" />
+                :class="[isAltColour ? [$style['c-nav-icon--alt']] : $style['c-nav-icon--gift']]" />
```
Comment (ashleynolan) [inline packages/components/organisms/f-header/src/components/Navigation.vue]: This is a little confusing atm – I'd always apply the `c-nav-icon--gift` class to the element and then the modifier class just gets applied based on `isAltColour`.

The styles that get applied by `c-nav-icon--alt` should overwrite the fill colour applied by the defaults set by `c-nav-icon--gift` as long as they come after them in the source order.

File: packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue (hunk 5 above)
Comment (FayeCarter) [reply packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue]: Ah yeah, this definitely makes no sense.
The `goBackIcon` is the orange arrow to close the country list when it's open.
The `backArrow` is the new chevron to open the country list. It should probably be `openCountriesIcon` or something.

Comment (ashleynolan) [reply packages/components/organisms/f-header/src/components/CountrySelectorPanel.vue]: Ok, cool. Could call it `c-countrySelector-closePanelIcon` and `c-countrySelector-openPanelIcon` to keep it pretty generic

**Review by jamieomaguire — DISMISSED** [review]

**Review by zlatinivanov-je — DISMISSED** [review]
LGTM
Just a note on the design, is it that only some nav items have an icon? e.g. Gift and Help are getting one but no others.

**Review by zlatinivanov-je — DISMISSED** [review]

**Comment by FayeCarter** [comment]
> 

So the other nav items, delivery and account, already had icons. Account was the only one that didn't so that gets a new icon and gift gets updated

**Review by jamieomaguire — APPROVED** [review]

**Review by charliearlie — APPROVED** [review]

**Review by zlatinivanov-je — COMMENTED** [review]
LGTM

**Pull request merged**